packages/react-day-picker/src/hooks/useModifiers/useModifiers.ts

```diff
@@ -21,9 +21,7 @@ export function useModifiers(date: Date): UseModifiers {
 
   // TODO: modifiers from context should not be overridden
```

Comment:
FWIW, I think the only modifiers that need overriding now are `disabled`, `range_start`, `range_middle`, and `range_end`.

Pretty sure these lines:
```
    modifiers.selected = modifiers.selected.concat(
      singleSelect.modifiers.selected ?? []
    )
```
can all now be deleted and still work

Reply:
(out of scope here I think though)

Reply:
@tjfred35 please remove it then :) Thanks!

Reply:
Err! Ok...does that mean a big refactor is cool?

Reply:
Alright...well...hope that wasn't premature, but I took a chance and made the big refactor. Based on some of the refactors you have been doing, I figured you would be comfortable reviewing it.